syntax = "proto3";
package proto;
option java_package = "sd.ufc.reserva.model";
option java_outer_classname = "ObjectReserva";

message Reserva{
    int32 id = 1;
    int32 id_usuario =2;
    int32 id_sala = 3;
    string data = 4;
    string horario = 5; 
}
